content/en/platform/corda/5.0/application-networks/managing/member-suspension/_index.md

```diff
@@ -24,7 +24,7 @@ The commands shown in this section, use the following variables:
 * `REST_API_PASSWORD` — your password for invoking the REST API.
 * `AUTH_INFO` — the authentication information if using PowerShell. You can set this as follows:
    ```
-   $AUTH_INFO = [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(("admin:admin" -f $username,$password)))
```

Comment:
Should we use admin/admin as the values for username and password when setting environment variables?

We need to tell the end user what the default values are somewhere. Do we have a page that tells people how to change them to something secure when setting up for the first time?

Reply:
No, we probably shouldn't ;) 
I think this is where they set up the credentials: https://corda5.preview.docs.r3.com/en/platform/corda/5.0/deploying-operating/deploying/deploying.html#database